jest-preset.json

```diff
@@ -7,12 +7,15 @@
     ]
   },
   "moduleNameMapper": {
-    "^[./a-zA-Z0-9$_-]+\\.(bmp|gif|jpg|jpeg|png|psd|svg|webp)$": "RelativeImageStub",
```

Comment:
Removing this broke importing files with multiple device resolution variations. Metro bundler will resolve `require('./file.png')` to `./file@2x.png` if you're on a 2x screen.

Reply:
Thanks for reporting! Could you give a bit more detail what the problem is? Apparently this change has broken your tests. Could you give some steps to reproduce?

I immediately notice that the regex `^[./a-zA-Z0-9$_-]+\\.(bmp|gif|jpg|jpeg|png|psd|svg|webp)$` does not match the `@` character in `./file@2x.png`. The regex has been moved from `moduleNameMapper` to the `transform` mapper on line 17 in the same file. Perhaps the change from `./file.png` to `./file@2x.png` happens between the mappers. @oblador, could you edit the regex on line 17 to `^[./a-zA-Z0-9@$_-]+\\.(bmp|gif|jpg|jpeg|png|psd|svg|webp)$`, so that it also matches `./file@2x.png`?

Reply:
This also breaks tests completely on Windows for me. Please see #19370 for details